Alert.c

```diff
@@ -0,0 +1,31 @@
+#include "Alert.h"
+#include "Config/ParameterConfig.h"
+#include <stdio.h>
+
+char *AlertType_str[AlertType_TotalNumber] =
+#if(BMS_ALERT_LANGUAGE == BMS_ALERT_LANGUAGE_ENGLISH)
```

Comment:
Choice of going to compiler switches:
Pro: It ensures that unnecessary stuff is not in the deliverable
Cons: 
- It needs multiple builds to be made and managed for each version of the software.
- Testing for 'missing translations' needs the entire program to be tested.

Any ideas to keep the pro without the cons?

Reply:
As far i know with c , no.
But is there any language that gives us such a feature ?(by keeping the pros and removing the cons )

Reply:
As discussed, could keep the texts in a separate txt file, swapping it for other languages at the time of deployment. Possible in PC, ESP32 and Raspberry PI even. But not possible in some embedded environments.